src/components/relatedProducts/outfit/OutfitCard.jsx

```diff
@@ -24,6 +24,21 @@ const ImageStyle = styled.img`
   object-fit: cover;
 `;
 
+const InnerDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  min-width: 300px;
+  justify-content: space-between;
+`;
+
+const IconStyle = styled.i`
```

Comment:
You can remove your commented lines of code if you don't need them - if you're keeping them in there for reference thats fine too though